### DISCUSSION TITLE
DSD playback through WASAPI instead of ASIO

### DISCUSSION THREAD

**Discussion by TT-ReBORN** [TT-ReBORN/Georgia-ReBORN#137] [General]
Original post by @Meisfuture, moved from Beta testing thread:

---

Windows native WASAPI interface hard DSD

The DSD dop mode of the Windows native lossless audio interface (WASAPI) is more stable than the DSD dop/native mode of the ASIO interface, and there is no popping at all. It is not necessary to select a third-party ASIO interface in order to use DSD native mode.

On the FOOBAR 2000 official website ASIO plug-in download page, it is also recommended not to use ASIO to play music.
<img width="891" alt="ASIO" src="https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/62739822-bd36-4b10-81d4-afea47b14634">

With a few simple settings, Foobar2000 can be used to call the Windows native WASAPI interface to hard solve DSD and play music in various other supported formats.
<img width="565" alt="2" src="https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/91a1caeb-410c-40ec-b5c4-dcf3f9c7a9df">
<img width="555" alt="3" src="https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/4f13695b-651d-43ab-a62c-6a85885c1df5">
<img width="565" alt="4" src="https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/102a3470-c74c-4c33-bf51-4b9bd6f9f022">
<img width="565" alt="5" src="https://github.com/TT-ReBORN/Georgia-ReBORN/assets/133914863/46fbec90-f45e-4ec6-9595-ced2bc74f30c">

After the setting is completed, Foobar2000 can not only call the audio device to hard decipher the DSD, but also call the audio device to play music of various other supported formats with high quality.

**Comment by d3rb** [comment]
this is useful but not the real solution for me. WASAPI Mode (DoP) DSD over PCM limited to DSD256!! [Hardware Limit of my Oppo Sonic Dac] For DSD512 Playback you need ASIO Mode
